pauthabielf64/pauthabielf64.rst

```diff
@@ -1117,80 +1120,80 @@ The GOT entries must be relocated by AUTH variant dynamic relocations.
   +-------------+----------------------------------------+----------------------------------------+--------------------------+
   | ELF 64 Code | Name                                   | Operation                              | Comment                  |
   +=============+========================================+========================================+==========================+
-  | 0x8110      | R\_AARCH64\_AUTH\_MOVW\_GOTOFF\_G0     | G(ENCD(GDAT(S))) - GOT                 | Set a MOV[NZ] immediate  |
+  | 0x245       | R\_AARCH64\_AUTH\_MOVW\_GOTOFF\_G0     | G(ENCD(GDAT(S))) - GOT                 | Set a MOV[NZ] immediate  |
```

Comment:
Very much a nit, but for the sanity of people that cross reference aaelf64 and pauthabielf64 documentation, it'd be nice if the ELF code in this doc wasn't written in hex.

Reply:
yea, sounds good to me.